conda/recipes/cudf/meta.yaml

```diff
@@ -30,7 +30,7 @@ requirements:
   run:
     - python
     - pandas>=0.24.2,<0.25
-    - cupy >=6.5.0,<7
+    - cupy >=6.6.0,<8.0.0
```

Comment:
This needs to be `>=6.6.0,<8.0.0a0` in case there are any pre-release versions published.

Reply:
In conda-forge we publish pre-releases to a separate label. So users would need to opt-in. Though we don't necessarily do this with every package. It's actually kind of rare. That said, happy to add the constraint.